### PR TITLE
add arktoshi to ark filter, rather than using a constant and dividing

### PR DESCRIPTION
**Pull request by ckhatri**
As per @luciorubeens suggestion in #407 

Created a filter to convert arktoshi into ark.

### CODE REVIEW

File: client/app/src/filters/filters.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ;(function () {
   'use strict'
 
-  angular.module('arkclient.filters')
+  angular.module('arkclient.filters' )
```
Comment (spresnal) [inline client/app/src/filters/filters.js]: Does the linter flag that additional space? (Also, on line 1 can you remove that semicolon -- unless it's necessary for some reason)

Comment (ckhatri) [reply client/app/src/filters/filters.js]: I'm not sure if the semicolon is necessary, but it's in most of the files so I think it's on purpose for some reason.

File: client/app/src/accounts/account.service.js [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
     self.defaultFees = {
       'send': 10000000,
-      'vote': 100000000,
+      'vote': ARKTOSHI_UNIT,
```
Comment (luciorubeens) [inline client/app/src/accounts/account.service.js]: This is the voting fee, it's not related to arktoshi

**Comment by luciorubeens** [comment]
merged manually +5 👍🏻

**Pull request merged**